src/extra/themes/default/lv_theme_default.c

```diff
@@ -955,8 +955,15 @@ static void theme_apply(lv_theme_t * th, lv_obj_t * obj)
     else if(lv_obj_check_type(obj, &lv_msgbox_class)) {
         lv_obj_add_style(obj, &styles->card, 0);
         lv_obj_add_style(obj, &styles->msgbox_bg, 0);
+        lv_obj_set_style_flex_main_place(obj, LV_FLEX_ALIGN_START, 0);
```

Comment:
In the theme, I'd prefer adding only style. Local style needs more RAM and can't be overwritten by "normal" styles later. 

However doesn't it work to use  the default `LV_FLEX_ALIGN_START` on each align?

Reply:
I see. Default style should work. Let me try that one.